mksnapshot.sh

```diff
@@ -9,41 +9,43 @@
 
 echo "RUN: $0"
 if [ $# -lt 1 ]; then
-	echo "Usage: $0 [branch]"
-	echo "Where [branch] is the path under /bzr/ to the branch to snapshot."
+	echo "Usage: $0 [branch] [tag]"
+	echo "Where [branch] is the name of the branch to snapshot."
+	echo "Where [tag] is the name of the tag on that branch to snapshot."
```

Comment:
So you are laughing at me for thinking this thing often mentioned in the git documentation as The "tag" to use for certain things and recommended for use in parameters which take tags, actually is a special-case tag? very funny.

I've tried going back to the bzr way of doing it, and found my initial mistake. bzr took the full path of the target branch, git only takes the name and assumes it is part of the current repository. That is sufficient for this scripts needs and is far simpler - so going with that now. Please review latest changes.